Packs/FeedUnit42v2/Integrations/FeedUnit42v2/FeedUnit42v2.py

```diff
@@ -0,0 +1,693 @@
+    """
+    Split indicator name into MITRE ID and indicator value: 'T1108: Redundant Access' -> MITRE ID = T1108,
+    indicator value = 'Redundant Access'.
+    """
+    ind_name = attack_indicator.get('name')
+    idx = ind_name.index(':')
+    ind_id = ind_name[:idx]
+    value = ind_name[idx + 2:]
+    return ind_id, value
+
+
+def add_stix_prefix_to_indicator(indicator: dict):
+    kill_chain_phases = indicator['fields']['killchainphases']
+    del indicator['fields']['killchainphases']
+    description = indicator['fields']['description']
+    del indicator['fields']['description']
+
+    indicator_type = indicator['type']
+    indicator['type'] = f'STIX {indicator_type}'
+    indicator['fields']['stixkillchainphases'] = kill_chain_phases
+    indicator['fields']['stixdescription'] = description
+
+    return indicator
+
+
+def create_attack_pattern_indicator(attack_indicator_objects, feed_tags, tlp_color, is_up_to_6_2) -> List:
+    """Parse the Attack Pattern objects retrieved from the feed.
+
+    Args:
+      attack_indicator_objects: a list of Attack Pattern objects containing the Attack Pattern.
+      feed_tags: feed tags.
+      tlp_color: Traffic Light Protocol color.
+      is_up_to_6_2: is the server version is up to 6.2
+
+    Returns:
+        A list of processed Attack Pattern.
+    """
+
+    attack_pattern_indicators = []
+
+    for attack_indicator in attack_indicator_objects:
+
+        publications = get_indicator_publication(attack_indicator)
+        mitre_id, value = get_attack_id_and_value_from_name(attack_indicator)
+
+        kill_chain_mitre = [chain.get('phase_name', '') for chain in attack_indicator.get('kill_chain_phases', [])]
+        kill_chain_phases = [MITRE_CHAIN_PHASES_TO_DEMISTO_FIELDS.get(phase) for phase in kill_chain_mitre]
+
+        indicator = {
+            "value": value,
+            "type": ThreatIntel.ObjectsNames.ATTACK_PATTERN,
+            "score": ThreatIntel.ObjectsScore.ATTACK_PATTERN,
+            "fields": {
+                'stixid': attack_indicator.get('id'),
+                "killchainphases": kill_chain_phases,
+                "firstseenbysource": handle_multiple_dates_in_one_field('created', attack_indicator.get('created')),
+                "modified": handle_multiple_dates_in_one_field('modified', attack_indicator.get('modified')),
+                'description': attack_indicator.get('description'),
+                'operatingsystemrefs': attack_indicator.get('x_mitre_platforms'),
+                "publications": publications,
+                "mitreid": mitre_id,
+                "reportedby": 'Unit42',
+                "tags": [tag for tag in feed_tags],
+            }
+        }
+        indicator['fields']['tags'].extend([mitre_id])
+        if tlp_color:
+            indicator['fields']['trafficlightprotocol'] = tlp_color
+
+        if not is_up_to_6_2:
+            indicator = add_stix_prefix_to_indicator(indicator)
```

Comment:
```suggestion
            indicator = change_attack_pattern_to_stix_attack_pattern(indicator)
```
because this is only done to one indicator type and it dose more than just adding a prefix.

Reply:
done

Reply:
and yes, in Unit42 it's just the attack pattern